parquet-hadoop/pom.xml

```diff
@@ -203,6 +209,12 @@
       <version>${commons-lang3.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.locationtech.jts</groupId>
+      <artifactId>jts-core</artifactId>
+      <version>${jts.version}</version>
+      <scope>test</scope>
```

Comment:
Why jts-core has been added twice with different scopes?

Reply:
Only test scoped jts-core is under the parquet-hadoop pom, the other one (w/o test scope) is under the parquet-column pom. If i removed the test scope, it errors out: 
```
[ERROR] Non-test scoped test only dependencies found:
[ERROR]    org.locationtech.jts:jts-core:jar:1.19.0:compile
```
I think it suggests that the jts-core dependency is included in the compile scope in parquet-column, but it's  being used in test code here in parquet-hadoop.